config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -776,6 +848,78 @@ fields:
               region: content
               weight: 100
 
+    -   name: researchstudy_projectdescription
+        content_type: research_study
+        label: Project Description
+        type: chado_property_type_default
+        description: Description of a project.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: local
+            termAccession: Project Description
+        display:
+          view:
+            default:
+              region: content
+              label: above
+          form:
+            default:
+              region: content
+
+    -   name: researchstudy_research_outputs
+        content_type: research_study
+        label: Research Outputs
+        type: chado_property_type_default
+        description: Include non-dataset focused research outputs here such as software and presentations.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: CO_010
+            termAccession: '0000159'
+        display:
+          view:
+            default:
+              region: content
+              label: hidden
+          form:
+            default:
+              region: content
+
+    -   name: researchstudy_url_token
+        content_type: research_study
+        label: Url-token
+        type: chado_property_type_default
+        description: This token should be unique and should not include spaces or any special characters. Usually it is based on the title or uniquename of the content. IT SHOULD NOT BE CHANGED ONCE SET OR YOU RISK BREAKING LINKS. If you must change it then make sure to setup a redirect for the old link.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: local
+            termAccession: 'url-token'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+          form:
+            default:
+              region: content
+
```

Comment:
This is a legacy field that we won't want to include here. Make sure to also remove it's term from the other yaml.